src/constants.ts

```diff
@@ -85,3 +85,85 @@ var resources: any = {
 };
 
 _.set(window, 'OPENSHIFT_CONSTANTS.CATALOG_HELP_RESOURCES', resources);
+
+var showAllCatalogs = function() {
+  $('body').find('.services-view-container ul li:nth-child(1) a')[0].click();
+};
+
+var showCategories = function() {
+  $('body').find('.services-view-container ul li:nth-child(2) a')[0].click();
+  setTimeout(function() {
+    $('body').find('.services-sub-category-tab')[1].click();
+  });
+};
+
+var homePageTourSteps = {
+  id: "getting-started-tour",
+  steps: [
+    {
+      title: "Create Project",
+      content: "Projects allow you to organize and manage your content. Projects require a unique name, and optionally can include a display name, and description",
+      target: ".catalog-projects-summary-panel .create-button",
+      targetScrollElement: '.landing-side-bar',
+      placement: "left"
+    },
+    {
+      title: "Search Catalog",
+      content: "Search by name, description, keyword, or label to quickly locate items in the catalog that you want to add to your project.",
+      target: ".landing-search-area .landing-search-form",
+      placement: "bottom",
+      fixedElement: true
+    },
+    {
+      title: "Build Applications",
```

Comment:
This section is going to be empty OOTB, so having a step for it won't make sense.  I'd rather not override the default catalog tour in the console constants. I would prefer to just add this step in the environments where we plan to have SaaS offerings configured (like Online).